plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/MultiParentCompileHook.java

```diff
@@ -86,7 +85,8 @@ public void validate(Map<String, Object> toCheck) throws Exception {
 
     @Override
     public boolean check(Map<String, Object> info) throws Exception {
-        return BlueOceanHook.isBOPlugin(info) || DeclarativePipelineHook.isDPPlugin(info);
+        return BlueOceanHook.isBOPlugin(info) || DeclarativePipelineHook.isDPPlugin(info)
```

Comment:
This is starting to get out of control, I do not really like the idea of having to add here every plugin but I understand that is for a follow up pr so I have created [JENKINS-49367](https://issues.jenkins-ci.org/browse/JENKINS-49367)